nucleus/admin/launcher/src/main/java/com/sun/enterprise/admin/launcher/GFLauncher.java

```diff
@@ -1,7 +1,7 @@
 /*
  * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
  *
- * Copyright (c) 2008-2015 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2008-2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Only a question: why you updated this copyright year?

Reply:
I missed the one already in place, hidden at https://github.com/payara/Payara/blob/285289a9684e5bcd9f3e712076bd5bafe190168e/nucleus/admin/launcher/src/main/java/com/sun/enterprise/admin/launcher/GFLauncher.java#L40

Have not revoked OCA, so it's still kind of true :wink: I think.

Reply:
@dmatej I'm going to check my pending open PRs.